src/styles/StudentStyles/index.less

```diff
@@ -0,0 +1,103 @@
+.student-dashboard {
+    background-color: lavender;
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+}
+
+.studentdb-left-column {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    //width: 50%;
```

Comment:
Let's remove commented-out code if it's no longer necessary!

Reply:
Code has been removed, FYI we're finishing up our work on the other PR: https://github.com/BloomTech-Labs/coder-heroes-fe/pull/216